go/tools/builders/cgo.go

```diff
@@ -29,6 +31,8 @@ import (
 	"unicode"
 )
 
+const linePrefix = "//line "
```

Comment:
This is largely personal preference (feel free to ignore), but I would move this next to the code that uses it. It would help to have the actual string be visible in that code; if I were editing this, I'd definitely be scrolling back to check whether there's a space at the end. I don't expect the string will be relevant for anything else at global scope.